page.js

```diff
@@ -0,0 +1,100 @@
+    detailed: 'https://www.linkedin.com/in/jamilaabass/?originalSubdomain=ke',
+  },
+  {
+    featuredImage: 'fsimages/Rana-el-Kaliouby-img.jpg',
+    Name: 'Rana el Kaliouby',
+    description: 'Rana el Kaliouby is an Egyptian-American computer scientist and entrepreneur.',
+    information: 'She specializes in the field of expression recognition research and technology development, which is a subset of facial recognition designed to identify the emotions expressed by the face',
+    detailed: 'https://www.linkedin.com/in/kaliouby/',
+  },
+  {
+    featuredImage: 'fsimages/emeagwali-img.jpeg',
+    Name: 'Philip Emeagwali',
+    description: 'A Nigerian computer scientist.studied for a Ph.D. degree from the University of Michigan from 1987 through 1991.',
+    information: 'He won the 1989 Gordon Bell Prize for price-performance in high-performance computing applications, in an oil reservoir modeling calculation using a novel mathematical formulation and implementation.',
+    More: 'https://www.linkedin.com/in/emeagwali',
+  },
+  {
+    featuredImage: 'fsimages/Prof-Omari.jpeg',
+    Name: 'Prof Omar Fakih',
+    description: 'Associate Professor of Computer and Telecommunications Engineering at the Engineering Faculty, Zanzibar University in Tanzania.',
+    information: 'Graduated PhD in Computer Engineering from Chonnam National University, South Korea. In 2000, he graduated Master of Engineering in Electronics and Communications specialized in Microwave from Birla Institute of Technology, India, where he also graduated his Bachelor of Electronics and Communications Engineering in July 1998.',
+    detailed: 'https://www.linkedin.com/in/omar-fakih-hamad-7b45844/?originalSubdomain=za',
+  },
+  {
+    featuredImage: 'fsimages/Tshilidzi-Marwala-img.jpeg',
+    Name: 'Prof Tshilidzi Marwala',
+    description: 'Vice-Chancellor and Principal of the University of Johannesburg, beginning January 2018.',
+    information: 'Tshilidzi Marwala is a South African artificial intelligence engineer, a computer scientist, a mechanical engineer and a university administrator.',
+    detailed: 'https://www.linkedin.com/in/tshilidzimarwala/?originalSubdomain=za',
+  },
+];
+
+// Render featured speakers data
+speakers.forEach((speaker) => {
+  const featured = `<div class="feature">
+    <div class="speaker-image">
+      <img src="${speaker.featuredImage}" width="500"
+      height="150" alt="speaker image"
+      
+      >
+    </div>
+    <div class="speaker-details">
+      <h4>${speaker.Name}</h4>
+      <p class="description">${speaker.description}</p>
+      <hr class="hr" hr></hr>
+      <p class=info>${speaker.information}</p>
+      <a target="_blank" class="detailed" href="${speaker.detailed}">More info</a>
+    </div>
+  </div>`;
+  featuredSpeakers.innerHTML += featured;
+});
```

Comment:
- [ ] [optional] It would have been nice to ensure the class feature has a fixed with and height so that your feature section matches the design given instead of some elements looking wider than the others